website/src/react-native/css/react-native.css

```diff
@@ -2030,6 +2030,14 @@ article li {
 .btn:hover {
   text-decoration: none !important; }
 
+@media screen and (max-width: 373px) {
+  .btn {
```

Comment:
Should I be more specific and add styles to only doc section? (I checked other pages  and they look good). I don't understand that wrong with margin-bottom in   `.buttons-unit .button`. Should I add `margin-bottom` only for the first button?